### PR TITLE
Manage students tab

### PR DESCRIPTION
**Pull request by caleybrock**
No new functionality, but data is now stored in redux.

### CODE REVIEW

File: apps/src/templates/teacherDashboard/sections.js [hunk 1]
```diff
@@ -71,6 +77,12 @@ export function renderLoginTypeAndSharingControls(sectionId) {
 }
 
 export function renderSectionTable(sectionId, loginType) {
+  registerReducers({manageStudents});
```
Comment (caleybrock) [inline apps/src/templates/teacherDashboard/sections.js:80]: I know we talked about adding `registerReducers` to index, but it looks like this file has two other places reducers are registered, so this seemed to make the most sense for now.

File: apps/src/templates/teacherDashboard/sections.js [hunk 2]
```diff
@@ -79,25 +91,13 @@ export function renderSectionTable(sectionId, loginType) {
...
-        username: student.username,
-        age: student.age,
-        gender: student.gender,
-        secretWords: student.secret_words,
-        secretPicturePath: student.secret_picture_path,
-        sectionId: sectionId,
-        loginType: loginType,
-      };
-    });
+    store.dispatch(
```
Comment (caleybrock) [inline apps/src/templates/teacherDashboard/sections.js:94]: I imagine the get request might get moved to redux if I need to have some sort of waiting UI.

File: apps/src/templates/manageStudents/manageStudentsRedux.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  }
+  if (action.type === SET_STUDENTS) {
+    return {
+      ...state,
+      studentData: action.studentData,
+    };
+  }
+
+  return state;
+}
```
Comment (Bjvanminnen) [inline apps/src/templates/manageStudents/manageStudentsRedux.js:36]: Would be good to also include some reducer tests (even tho this is clearer pretty simple so far)

File: apps/src/templates/teacherDashboard/sections.js (hunk 1 above)
Comment (Bjvanminnen) [reply apps/src/templates/teacherDashboard/sections.js:80]: This seems okay.

**Comment by caleybrock** [comment]
👍 added some minimal test coverage for the reducer

File: apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  const initialState = manageStudents(undefined, {});
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers({manageStudents: manageStudents});
+  });
+
+  afterEach(() => {
+    restoreRedux();
+  });
```
Comment (Bjvanminnen) [inline apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js]: I don't think you need any of the before/after here. You're not really using redux at all - just testing the reducer directly.

**Pull request merged**